src/modules/endpoint/ContentLoadCriterion.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import moment from 'moment-timezone'
+import Moment from 'moment'
+
+const MAX_CONTENT_AGE = 24
+
+export type ContentLoadCriterionType = {|
+  peek?: boolean,
```

Comment:
Is peek really a content load criterion and not a property which some screen should open with? :thinking:

Reply:
I renamed the functions a bit